rust/generated_parser/src/early_errors.rs

```diff
@@ -73,71 +74,103 @@ impl<'alloc> IdentifierEarlyErrorsContext<'alloc> {
     }
      */
 
+    fn is_arguments(terminal_id: TerminalId, name: Name<'alloc>) -> bool {
```

Comment:
This will sound clunky, but rename these methods to `is_arguments_identifier` etc. Otherwise, `is_yield` sounds like it checks for the `yield` *keyword*, which is not what it checks.